packages/components/src/Card/Card.css

```diff
@@ -64,17 +60,9 @@
 .header {
   display: flex;
   padding: var(--card--base-padding);
+  padding-bottom: 0;
   background-color: var(--card--header-background);
```

Comment:
We could in theory just delete this declaration and the variable for the card header background now.